rust-toolchain

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.63"
+channel = "1.65"
```

Comment:
Do not resolve:
- need to squash non-main commits so as not to pollute version history